.ci/check_source_file_changes.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { xrun } from "../prettier/util.ts";
+import { red, green } from "../colors/mod.ts";
+
+/**
+ * Checks whether any source file is changed since the given start time.
+ * If some files are changed, this function exits with 1.
```

Comment:
Probably should describe the glob behavior here.

Reply:
I'm not sure I follow. There's no glob in this file.

Reply:
Oops. I left this comment in the wrong place.